tests/e2e/support/objects/app-admin-settings/users/actions.ts

```diff
@@ -246,6 +246,10 @@ export const filterUsers = async (args: {
   values: string[]
 }): Promise<void> => {
   const { page, filter, values } = args
+  if (await page.locator(closeEditPanel).count()) {
+    // close potentially open sidebar panel to avoid focus jumping
+    await page.locator(closeEditPanel).click()
+  }
```

Comment:
Can't reproduce that locally, just happens in the tests... 🤷 So IMO nothing to worry about in the "real world".